screens/CameraRecording.tsx

```diff
@@ -65,19 +66,21 @@ const SavedOverlayViews = StyleSheet.create({
   },
 });
 
-const COUNTDOWN_TIME = 5;
+const COUNTDOWN_TIME = 10;
 // ~150MB
 const MAX_FILE_SIZE = 150000000;
 const FourEightyP = "480P";
 
 export interface CameraRecordingProps {
   overlay: ReactElement;
   finished: () => void;
+  name: string;
```

Comment:
Sorry to nitpick but Can we rename this to like videoName? It's a little more descriptive of what the prop is used for within the component

Reply:
What a RIDICULOUS idea! (yes that is actually a MUCH better name than name)